lib/reverse_string.rb

```diff
@@ -1,5 +1,17 @@
 def reverse_characters(str)
+    newarr = []
+	str = str.split("")
+	str.length.times do
+		newarr.push(str.pop)
+		end
+	return newarr.join
 end
 
 def reverse_words(str)
+    newarr = []
+	str = str.split()
+	str.length.times do
+		newarr.push(str.pop)
+		end
```

Comment:
Yeah, I have noticed that, for some reason, my line indentations in C9 don't always translate over to Github perfectly. I'll keep an eye out for that!